armory/data/datasets.py

```diff
@@ -295,6 +295,30 @@ def _parse(serialized_example):
     return clean_x, adv_x, labels
 
 
+def imagenette(
+    split_type: str,
+    epochs: int,
+    batch_size: int,
+    dataset_dir: str = None,
+    preprocessing_fn: Callable = None,
+) -> ArmoryDataGenerator:
+    """
+    Smaller subset of 10 classes of Imagenet
+        https://github.com/fastai/imagenette
+    """
+    if batch_size != 1:
+        raise NotImplementedError("Due to variable length input, batch_size must be 1")
```

Comment:
At some point I think we'll need to add a reshape argument to standardize variable image shapes for some datasets. (audio not as simple).